### DISCUSSION TITLE
Bunches have the same name in menu bar

### DISCUSSION THREAD

**Discussion by vw-github** [ttscoff/bunch-docs#334] [Q&A]
I have just started with Bunch and have created 2 bunches with different names:

<img width="459" alt="Screenshot 2024-08-12 at 14 07 44" src="https://github.com/user-attachments/assets/dac7208a-3d96-4ec1-837d-eaf46021088c">

But in the menu bar they appear with the same name:

<img width="321" alt="Screenshot 2024-08-12 at 14 06 33" src="https://github.com/user-attachments/assets/c8b9d633-c10d-46dd-ae42-8e39e5f933b0">

macOS 14.6.1, M2 MacBook Air. Bunch 1.4.17.

Any help with this issue would be welcome.

**Comment by ttscoff** [comment]
You'll see a title: key at the top of each of your Bunches. Change that 
appropriately.

**Comment by vw-github** [comment]
Change title and now fixed. Thank you very much.
